pkg/model/coordinator/coordinator.go

```diff
@@ -163,8 +157,10 @@ func (coo *Coordinator) InitState(bootstrap bool, startIndex milestone.Index) er
 			return fmt.Errorf("previous milestone does not match latest milestone in database: %d != %d", startIndex-1, latestMilestoneFromDatabase)
 		}
 
-		// if we bootstrap a network, NullHash has to be set as a solid entry point
-		tangle.SolidEntryPointsAdd(hornet.NullHashBytes, startIndex)
+		if startIndex == 1 {
```

Comment:
I still think this is wrong. You can bootstrap the network at any index. e.g. after someone decides to do a global snapshot add add a gap in the milestones

Reply:
But then I don't want to add NullHashBytes as a solid entrypoint. COO should reference the last issued milestone even with gaps.... (but why add gaps at all :scream: )